server/aesd_ioctl.h

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
Shouldn't need to keep a separate copy of this file, would be better to share with the driver.